src/constants.js

```diff
@@ -0,0 +1,6 @@
+export const DEFAULT = {
+  SHOW_HOME_MESSAGE: false,
```

Comment:
I think it makes code less readable, but nothing blocking and depends on preferences.

Reply:
I talked about this approach with Damian and Seweryn, and they said it might be a good idea, so I went ahead and did it like that.

Reply:
This allows to store all default configuration variables in one place.

Reply:
yes, @olewandowski1 started to implement such aspects in this new way. @wzglinieckisoldevelo